website/pages/docs/plugins/sources/gitlab/tables.md

```diff
@@ -6,5 +6,6 @@
   - [gitlab_group_members](https://github.com/cloudquery/cloudquery/blob/main/plugins/source/gitlab/docs/tables/gitlab_group_members.md)
 - [gitlab_projects](https://github.com/cloudquery/cloudquery/blob/main/plugins/source/gitlab/docs/tables/gitlab_projects.md)
   - [gitlab_projects_releases](https://github.com/cloudquery/cloudquery/blob/main/plugins/source/gitlab/docs/tables/gitlab_projects_releases.md)
+  - [gitlab_project_branches](https://github.com/cloudquery/cloudquery/blob/main/plugins/source/gitlab/docs/tables/gitlab_project_branches.md)
```

Comment:
`gitlab_projects_releases` vs `gitlab_project_branches`? A bit inconsistent... I would recommend doing `gitlab_<singular>_<plural>` but that would mean a breaking change (rename `projects_releases` to be `project_releases`)

Reply:
Agreed. I saw that and will fix it in a future PR that has a breaking change.